src/components/Article.jsx

```diff
@@ -0,0 +1,31 @@
+import { fetchAllArticles } from "../utils/endpoints"
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+function ArticleList() {
+    const [articleList, setArticleList] = useState([]);
```

Comment:
Please use a loading state and message rendered in the page while articles are loading.